src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/Content/Header/TitleBlock/Actions/_update.html.twig

```diff
@@ -1,3 +1,9 @@
 {% import '@SyliusAdmin/Shared/Helper/button.html.twig' as _button %}
 
-{{ _button.primary(sylius_test_form_attribute('update-changes-button')|merge({ text: 'sylius.ui.update'|trans, type: 'submit', form: hookable_data.form.vars.id })) }}
+{{ _button.primary(
+    sylius_test_form_attribute('update-changes-button')|merge({
+        text: 'sylius.ui.update'|trans,
+        type: 'submit',
+        form: hookable_metadata.context.form.vars.id
+    })
```

Comment:
We really should reverse this merge, it looks weird having test attributes as the base.

Reply:
Or we should a better way to add test form attributes :D.